SYCL/ESIMD/imulh_umulh.cpp

```diff
@@ -11,7 +11,7 @@
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 
 // TODO: the test gives irregular fails in CI (flaky), disabled until resolved.
-// XFAIL: *
+// UNSUPPORTED: gpu
```

Comment:
BTW, the test fails on ESIMD emulator as well, so I don't know if this is enough to fix all issues in pre-commit CI.

Reply:
That is odd. With most fresh SYCLOS compiler and esimd-emulator, I have 1000 passes out of 1000 runs locally.
Can you give some link to CI fail with emulator?

Reply:
Thanks for the update. Let's wait for the CI to pick up this change to see if it works.